packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -134,5 +305,31 @@ export default class {
       targetEntityId: this.targetEntityModel.entityId,
       removedById: removedBy,
```

Comment:
For consistency with some of the other change requests for account IDs. (there's a param jsdoc as well)
```suggestion
      removedById: removedById,
```

Reply:
https://github.com/hashintel/hash/pull/1081/commits/078568b93b61f7b3a0cb85e506f5f08328b89b0c